code/game/machinery/computer/store.dm

```diff
@@ -39,6 +39,12 @@
 
 /obj/machinery/computer/merch/New()
 	..()
+	var/month = time2text(world.realtime,"MM")
+	var/day = time2text(world.realtime,"DD")
+	if(month == "02" && day == "14")
```

Comment:
Might be an idea to have a define or macro or something, so we don't have all these month and day variables floating around, and encourage others to have some things locked behind certain dates.